generators/constants.cjs

```diff
@@ -64,8 +64,6 @@ module.exports = {
     javassist: '3.27.0-GA', // Should match Hibernate deps
     javaxMail: '2.0.1',
     jbcrypt: '0.4',
-    liquibase: '4.27.0',
-    liquibaseHibernate6: '4.27.0',
     logback: '1.5.3',
```

Comment:
Micronaut 4.3.8 is the latest version, but we can update to that in another PR.

Reply:
Created PR at https://github.com/jhipster/generator-jhipster-micronaut/pull/335.